### ISSUE TITLE
[Feature] Import libs from package.json

### ISSUE THREAD

**Issue by max-mykhailenko** [unlight/sublime-import-helper#20]
Hi, can I ask you to write one more feature? I want to import `react`, `lodash` (all and as separate functions) and other modules from `package.json` dependencies and dev dependencies.

Is it possible?

Thanks

**Comment by unlight** [comment]
I'm not sure that I understand the request. Can you give an example?

> other modules from package.json dependencies and dev dependencies.

It's already (see screenshot in readme, e.g. '@angular/http')
If you cant find something which are defined in dependencies (or dev) that's maybe another bug.

**Comment by max-mykhailenko** [comment]
When I want to import lodash I got this error.

<img width="399" alt="screen shot 2017-03-24 at 3 08 39 pm" src="https://cloud.githubusercontent.com/assets/1213198/24295529/cab9affc-10a3-11e7-94d4-4258445a33d4.png">

Part of ST settings, maybe this blocks from parsing
```json
	"folder_exclude_patterns":
	[
                ...
		"node_modules",
```

In imports list I also don't see `lodash`.

**Comment by unlight** [comment]
That's because `lodash` is written in ES5, import-helper uses `esm-exports` under the hood, and it can parse only ES6 modules / TypeScript definition files.
You can try `lodash-es` instead.

Second option is parse typings for lodash-es5 `@types/lodash`, but it's not implemented yet. I've created issue [esm-exports#7](https://github.com/unlight/esm-exports/issues/7)

**Comment by unlight** [comment]
Checked `lodash` and `@types/lodash`
From `lodash` module, I got such response:
```
 Entry {
    name: undefined,
    specifier: undefined,
    isDefault: false,
    module: 'lodash',
    filepath: undefined,
    cjs: true,
    ts: undefined }
```
It's possible to insert import, but it depends on #13 #14 
Similar functionality has [EasyRequire](https://packagecontrol.io/packages/EasyRequire) package.
`@types/lodash` is failed to parse, it uses triple directive `/// <reference ...` it must be parsed by typescript.

Result of parsing `lodash-es` is good: 200+ imports found.
So basically, issue can be closed because of ES5 `lodash` nature, see [comment](https://github.com/unlight/sublime-import-helper/issues/20#issuecomment-289081001).

**Comment by unlight** [comment]
Imlemented in version 2.1.0, non esm modules will be imported as 'import all' `import * as lib` or 'import default'
